### DISCUSSION TITLE
New note : journal AND inbox

### DISCUSSION THREAD

**Discussion by davidszerman** [TriliumNext/Trilium#3280] [Q&A]
Hello,

I love the two concepts that you offer...

- The inbox for new notes
- The new note added in the calendar subtree

But as far as I have seen, the calendar subtree is used if no #inbox parent note but what if I would like to use both features : 

- the note added to the date it was created : useful for a journaling note, to follow what happened,...
- the note Inbox as a clone in order to store the notes to be organized : if it is in the inbox, I have to move it to another subtree or delete it (journaling notes : I delete the clone in the inbox, keep a clone in the calendar)

Is it possible ? 

Thanks

**Comment by zadam** [comment]
It's possible with scripting - use "Today" script as a baseline, create the note first (using `runOnBackend()` under today's note and then `note.cloneTo(parentNoteId);` ...

**Reply by davidszerman** [reply]
Thanks for the idea but unfortunately I do not understand how to do this. I was a  computer student but it was a long time ago...

Which not should I create first ? The new one under the today note ?

Where should be the code ?

Sorry for my stupid questions but that it is not easy for me...

Trilium seems to be so powerful ! Scripting helps to improve the software ! But I have to understand how it works   ;-S

**Reply by davidszerman** [reply]
@zadam 

Could you please help me ?

Should I put this code into today button note ? In the template ?

I understand the code (pretty simple actually) but I do not get where to put it...

Thanks

**Reply by zadam** [reply]
> Could you please help me ?

I'm sorry, but no. To avoid drowning in support work, I have to set some boundaries, and writing custom scripts for users is beyond that boundary.
